### PR TITLE
Replace Logo component with a more idiomatic example

### PR DESCRIPTION
**Pull request by jaitaiwan**
Replaces the Logo component with a simple grommet Icon to prevent users from assuming there is a built in logo component with appropriate styling.

This addresses grommet/grommet/issues/1073

### CODE REVIEW

**Comment by jaitaiwan** [comment]
Build appears to be failing due to not being able to execute `eval yarn`

File: src/docs/components/login-form/LoginFormDoc.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default class LoginFormDoc extends Component {
             <dt><code>forgotPassword  {"{element}"}</code></dt>
             <dd>A link that would take the user to a new page.</dd>
             <dt><code>logo            {"{element}"}</code></dt>
-            <dd>A logo component.</dd>
+            <dd>A react node. Best suited to an svg element or font icon.</dd>
```
Comment (alansouzati) [inline src/docs/components/login-form/LoginFormDoc.js:46]: can we remove the font icon? I don't think this is something we want to recommend. Also would you mind fixing the conflicts?

Sorry for taking such a long time to get back to you on this.

**Review by alansouzati — CHANGES_REQUESTED** [review]

**Comment by tracybarmore** [comment]
Hey @jaitaiwan. Sorry for the delayed feedback on this PR. Would you mind updating the pull request to remove the conflicts so we can successfully merge? Thanks for your contribution!

**Comment by jaitaiwan** [comment]
I believe that with the requested changes my PR becomes pointless perhaps?

**Comment by alansouzati** [comment]
I don't think so @jaitaiwan 

how about

```
A react node. Best suited to an svg element or a custom component containing an svg element.
```

**Comment by alansouzati** [comment]
Fixed in https://github.com/grommet/grommet-docs/commit/3c7be98435a3670d61c09db5954112643edfa202

**Pull request closed without merging**